styles/variables.scss

```
@@ -6,3 +6,5 @@ $greyGreen: #87bba2;
 $background: #55828b91;
 //////////////////
 $height: calc(100vh - 80px);
+/////////////////
+$backgPic: url("/public/backg.svg");
```

The image doesn't work, because `/public/backg.svg` is an incorrect path. Everything inside of `/public` directory is available directly from the root of your website.
```suggestion
$backgPic: url("/backg.svg");
```